assets/queries/cloudFormation/docdb_cluster_master_user_password_rule/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "DocDB_Cluster_Master_User_Password_Rule",
+  "queryName": "DocDB Cluster Master User Password Rule",
+  "severity": "MEDIUM",
+  "category": null,
+  "descriptionText": "DocDB DB Cluster master user password must not be in a plain text string or referenced in a parameter as a default value.",
+  "descriptionUrl": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-docdb-dbcluster.html"
+}
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline